internal/providers/terraform/azure/testdata/frontdoor_test/frontdoor_test.golden

```diff
@@ -0,0 +1,25 @@
+
+ Name                                                   Monthly Qty  Unit            Monthly Cost 
+                                                                                                  
+ azurerm_frontdoor.frontdoor_with_usage                                                           
+ ├─ Routing rules (first 5 rules)                             3,650  hours                $109.50 
+ ├─ Routing rules (per additional rule)                         730  hours                  $8.76 
+ ├─ Frontend hosts (over 100)                                    10  hosts                 $50.00 
+ ├─ Inbound data transfer                                     1,000  GB                    $10.00 
+ └─ Outbound data transfer                                                                        
+    ├─ North America, Europe and Africa (first 10TB)         10,000  GB                 $1,700.00 
+    ├─ North America, Europe and Africa (next 40TB)          40,000  GB                 $6,000.00 
+    └─ North America, Europe and Africa (next 100TB)        150,000  GB                $19,500.00 
```

Comment:
FYI The issue lists `Outbound data transfer` as a first component, but subresources are pushed to the bottom.
Also, Pricing API includes pricing for "over 150TB" tier which the same as "next 100TB". Should we include an additional tier here?

Reply:
The pricing calculator uses the same price for >150 as well, but the pricing page says "Contact Us". I think we have two options here.
1. Have the last tier as `over 50TB` using the $0.13 price.
2. Have the `next 100TB` tier and a `over 150TB` tier that just use the same price.

Any thoughts @alikhajeh1?

Reply:
I vote for Option 1 as the golden file tests will catch if/when prices change for the last tier. So for now we can keep it simple and adjust later if needed.